sunpy/instr/aia/response.py

```diff
@@ -229,6 +229,7 @@ def get_wavelength_response(self, channel, use_response_table2=False, use_calc_e
         ev_per_angstrom = (12398.4953 * u.eV * u.angstrom)
```

Comment:
You can actually do all of this through the magic of `astropy.units` and `astropy.constants`. This is a bit more self-consistent and simultaneously provides an explanation for the "magic number" 12398
```Python
>>> hc = (constant.h).to(u.eV*u.s)*(constant.c).to(u.angstrom/u.s)
>>> hc
<Quantity 12398.419292004202 Angstrom eV>
```